features/support/apiREST.js

```diff
@@ -13,16 +13,16 @@ ApiREST.prototype.init = function (world) {
 ApiREST.prototype.disconnect = function () {};
 
 ApiREST.prototype.pathApi = function (path) {
-  return config.url + '/api/' + path;
+  return config.url + '/api/v1/' + path;
```

Comment:
:+1: API version number will be required by protocol plugins, to make sure that a protocol plugin will be compatible to a given version of Kuzzle.

It may be interesting to also have a minor version number though, like 'v1.0', to distinguish between minor and major changes.